syntax = "proto3";

import "adlinktech/datariver/descriptor.proto";

package com.adlinktech.example.protobuf;


message RotationalSpeed {
  option(.adlinktech.datariver.tag_group) = {
    qosProfile: "telemetry"
    description: "ADLINK Edge SDK Example Rotational speed TagGroup"
  };

  uint32 speed = 1 [(.adlinktech.datariver.field_options) = {
    description: "Rotational speed"
    unit: "rpm"
  }];
  uint32 lastHourMin = 2 [(.adlinktech.datariver.field_options) = {
    description: "Minimum rotational speed in last hour"
    unit: "rpm"
  }];
  uint32 lastHourMax = 3 [(.adlinktech.datariver.field_options) = {
    description: "Maximum rotational speed in last hour"
    unit: "rpm"
  }];
  float lastHourAverage = 4 [(.adlinktech.datariver.field_options) = {
    description: "Avarage speed in last hour"
    unit: "rpm"
  }];
};
